src/main/asciidoc/output/index.adoc

```diff
@@ -303,4 +303,60 @@ JUnit 5 allows multiple methods to exist for the same lifecycle events.
 As an example, it is possible to define 3 `@BeforeEach` methods on the same test.
 Because of asynchronous operations it is possible that the effects of these methods happen concurrently rather than sequentially, which may lead to inconsistent states.
 
-This is a problem of JUnit 5 rather than this module. In case of doubt you may always wonder why a single method can't be better than many.
\ No newline at end of file
+This is a problem of JUnit 5 rather than this module. In case of doubt you may always wonder why a single method can't be better than many.
+
+==== Support for RxJava 1 and 2 Vertx instances
+
+Reactive extension support in Vert.x is being provided by the `vertx-rx-java` and `vertx-rx-java2` modules.
+They provide shims / wrappers around the Vert.x core APIs, like `link:../../apidocs/io/vertx/rxjava/core/Vertx.html[Vertx]` (RxJava 1) and `link:../../apidocs/io/vertx/reactivex/core/Vertx.html[Vertx]` (RxJava 2).
+
+Instances of these _"Rx-ified"_ `Vertx` classes can be injected in test and lifecycle methods, as in:
+
+[source,java]
+----
+@DisplayName("👀 A RxJava 2 + Vert.x test")
+@ExtendWith(VertxExtension.class)
+class RxJava2Test {
+
+  @BeforeEach
+  void prepare(Vertx vertx, VertxTestContext testContext) {
+    RxHelper.deployVerticle(vertx, new ServerVerticle())
+      .subscribe(id -> testContext.completeNow(), testContext::failNow);
+  }
+
+  @Test
+  @DisplayName("🚀 Start a server and perform requests")
+  void server_test(Vertx vertx, VertxTestContext testContext) {
+    Checkpoint checkpoints = testContext.strictCheckpoint(10);
+
+    HttpRequest<String> request = WebClient
+      .create(vertx)
+      .get(8080, "localhost", "/")
+      .as(BodyCodec.string());
+
+    request
+      .rxSend()
+      .repeat(10)
+      .subscribe(
+        response -> testContext.verify(() -> {
+          assertThat(response.body()).isEqualTo("Ok");
+          checkpoints.flag();
+        }),
+        testContext::failNow);
```

Comment:
more a general comment. When using RX Java, you are probably going to use RX Java testing features.